package.json

```diff
@@ -74,6 +74,7 @@
 		"overrides": {
 			"@types/react": "17.0.x",
 			"@wordpress/data": "wp-6.0",
+			"@wordpress/keyboard-shortcuts": "wp-6.4",
```

Comment:
I faced an error during the execution of the tests with a lower version of this dependency.